### DISCUSSION TITLE
Looking for maintainer

### DISCUSSION THREAD

**Discussion by xupefei** [QL-Win/QuickLook#1474] [Announcements]
Hey everyone!

I wanted to address the recent lack of progress on this project. Over the past year, QuickLook development has stalled, and I apologize for neglecting your discussions and PRs. Regrettably, I can no longer maintain this project.

My available free time has significantly decreased compared to past years, but the primary reason for stepping down is that all my machines now run either Linux or macOS, leaving me without a Windows environment for development.

Therefore, I'm reaching out to the community to see if anyone with interest and capability would be willing to take over this project. The main responsibilities will be:

1. Review PRs;
2. Engage with the community regarding issues;
3. Code: fix bugs, catch up with Windows API changes, etc.

I could also highlight some essential knowledge for being a maintainer:

1. C# and WPF.
2. Win32 API.
3. Basic Windows debugging ability.

Maybe I should not require too much, but I would want the new maintainer to promise the following at least:

1. Be nice to the community;
2. Keep QuickLook [free and open-source](https://en.wikipedia.org/wiki/Free_and_open-source_software).

If you are interested, please leave a message below or reach out directly. Thank you 😊

Cheers,
Paddy

**Comment by rhzxg** [comment]
There is a repository called `PowerToys` maintained by Microsoft, which has a similar module inside. Is this repository related to that one?

**Reply by Dhyfer1** [reply]
No. Although the PowerToys Peek module is similar to Quicklook, it does not have support for most files like Quicklook does.

**Reply by edge261** [reply]
maybe Quicklook could be converted to Peek... since it's better than Peek.

**Reply by xupefei** [reply]
Peek is in its early stage, and when I tried it a while ago, I saw it depended on the Windows Explorer preview panel plugins rather than shipping its own plugins. I'm not a big fan of that.

**Comment by shanto** [comment]


**Reply by edge261** [reply]
tried to install your fork and I got this:
![image](https://github.com/user-attachments/assets/b440a864-1cbc-464f-bae2-adc01ba3a6bd)
maybe a false positive?

**Reply by xupefei** [reply]
Thank you very much for your interest. I reviewed your profile and noticed that your expertise is primarily on the front-end side, which does not align with the needs of this project. I hope you find projects that better fit your skills.

**Comment by rabelux** [comment]
I've seen some valuable contributions by @mooflu in the past so maybe give him developer rights?

@xupefei I'd like to shout out a big thank you for all you've put into this project. I'm still using it almost every day, still recommend it to friends/colleagues and even in the neglected state it still improves my productivity and I love it. Thanks for making this possible, thanks for making it open source and I wish you the best for your future!

Depending on the future of this project I can still offer to keep an eye on the issues. But as long as this project doesn't switch to Python I won't be of much help regarding coding ;)

**Reply by xupefei** [reply]
Thank you so much for the help over these years!

**Comment by mooflu** [comment]
FWIW, there's also a modified QL (not a direct github fork) by @emako here https://github.com/emako/QuickLook#modification

**Reply by rabelux** [reply]
All links still lead to the original repo, so I would have  to compile it myself?

**Reply by mooflu** [reply]
Posted mainly to ping the author to see this discussion in case they are interested. I have not used their version.

**Reply by emako** [reply]
I once thought QuickLook was no longer being maintained. 😢

**Reply by xupefei** [reply]
> QuickLook was no longer being maintained

True. Are you guys willing to take over?

**Reply by xupefei** [reply]
@shanto I replied to your comment in another thread.
Here I'm asking @mooflu and @emako to see if they do have the interest.

**Reply by emako** [reply]
@xupefei  I'm willing to take over, QuickLook works for me a lot.

**Reply by mooflu** [reply]
No, from me, but happy to contribute every now and then.

**Reply by xupefei** [reply]
@mooflu NP, thanks for your contributions!
@emako Wonderful. I'll give you the permission this weekend. Also I'll check how to transfer my Windows Store account to you, so you could publish new versions.

**Reply by emako** [reply]
> @mooflu NP, thanks for your contributions! @emako Wonderful. I'll give you the permission this weekend. Also I'll check how to transfer my Windows Store account to you, so you could publish new versions.

okay

**Reply by Dhyfer1** [reply]
@xupefei Before you go, could you please fix the FontViewer plugin? I know you didn't make that plugin, it's just that it's the only plugin on the list that isn't compatible with the latest version of Quicklook and I was wondering if you could fix this, as the original developer seems to be lost and has stopped responding to issues quite some time ago.

**Comment by xupefei** [comment]
The Windows Store app is now migrated to @emako.
Thanks @emako for being the new maintainer!
